GDFirebase/FirebaseAuth.gd

```diff
@@ -87,16 +97,17 @@ func begin_refresh_countdown():
     refresh_request_body.refresh_token = refresh_token
     request(refresh_request_url, ["Content-Type: application/json"], true, HTTPClient.METHOD_POST, JSON.print(refresh_request_body))
 
+# This function is used to make all keys lowercase
+# This is only used to cut down on processing errors from Firebase
```

Comment:
Can we add, "Due to inconsistencies in the Firebase REST API"? Want to make it clear that it's the Google team's fault. :P

Reply:
Added